Source/Scene/Model.js

```diff
@@ -343,7 +343,7 @@ define([
                     var result = parseBinaryGltfHeader(gltf);
 
                     // CESIUM_binary_glTF is from the beginning of the file but
-                    // KHR_binary_glTF is from the beginning of the binary section
+                    // binary_glTF is from the beginning of the binary section
```

Comment:
Technically the extension is still called KHR_binary_glTF so there's no reason to change these comments.